Khan/perseus
# Fix `code` styling in content editor (and occasionally in production)

## Summary:

@kmerrill27 noticed that in the content editor when previewing in Desktop mode, formatted code does not use a monospace font.

There are two reasons for this:

  1. There is a general `code` and `pre` rule in `khan-exercise.css` that should apply, but when Perseus is used in webapp, the webapp CSS reset is applied after, negating those styles.
  1. There is a desktop style for `code` and `pre` elements, but right now they are nested inside a `.perseus-mobile` rule, meaning they can't apply because we only apply that class in mobile contexts. :facepalm: (See [here](https://github.com/Khan/perseus/blob/9e522234f03bd9ae9fe5d4554827bb41012ee183/packages/perseus/src/styles/articles.less#L585)).

So, for now the simplest, least invasive fix is to hoist the common `code` and `pre` element styles scoped within `.framework-perseus`.  Other, existing styles for mobile and articles will still apply, but this will guarantee that we have a base style that has a higher specificity than any reset styles.

Issue: LC-1482

## Test plan:

  1. Navigate to https://www.khanacademy.org/devadmin/content/exercises/tracing-for-loops-with-range/xfb1f2616e5139982/x44d92fae4709f15d
  1. Tap the "Desktop" button in the right-hand preview
  1. Note that the radio widget options that include code are formatted width font
